File: csharp/GSDK_CSharp_Standard/Model/MaintenanceSchedule.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace Microsoft.Playfab.Gaming.GSDK.CSharp.Model
{
    public class MaintenanceSchedule
    {
        public string DocumentIncarnation { get; set; }

        public IList<MaintenanceEvent> Events { get; set; }
    }

    public class MaintenanceEvent
    {
        public string EventId { get; set; }

        public string EventType { get; set; }

        public string ResourceType { get; set; }

        public IList<string> Resources { get; set; }

        public string EventStatus { get; set; }

        public DateTime? NotBefore { get; set; }

        public string Description { get; set; }

        public string EventSource { get; set; }

        public int DurationInSeconds { get; set; }
    }
}

File: java/gameserverSDK/src/main/java/com/microsoft/azure/gaming/MaintenanceSchedule.java
package com.microsoft.azure.gaming;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class MaintenanceSchedule {
    
    private String documentIncarnation;

    @SerializedName(value = "Events")
    private List<MaintenanceEvent> events;

    public String getDocumentIncarnation() {
        return documentIncarnation;
    }

    public List<MaintenanceEvent> getMaintenanceEvents() {
        return events;
    }
}